deploy/003_deploy_optimism_spokepool.ts

```diff
@@ -15,17 +15,11 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     hubPool.address,
   ];
 
-  // Construct this spokepool with a:
-  //    * A WETH address of the WETH address
-  //    * A depositQuoteTimeBuffer of 1 hour
-  //    * A fillDeadlineBuffer of 6 hours
-  //    * Native USDC address on L2
-  //    * CCTP token messenger address on L2
```

Comment:
I've removed each of these for two reason:
- The input arguments are almost always symbolically named now.
- This comment is liable to get out of sync with the actual contract initializer. It's always better to check against the official source.